src/providers/execution/contracts/lido.py

```diff
@@ -1,12 +1,13 @@
 import logging
-from functools import lru_cache
 
 from eth_typing import ChecksumAddress
 from web3.types import Wei, BlockIdentifier
 
 from src.modules.accounting.types import LidoReportRebase, BeaconStat
 from src.providers.execution.base_interface import ContractInterface
 from src.utils.abi import named_tuple_to_dataclass
+from src.utils.cache import global_lru_cache as lru_cache
```

Comment:
After discussion with @F4ever it is clear that this fix might be ussefull in case if for next oracle cycle run we have cached values for `latest` or other types which should not be cached in that case.